### PR TITLE
Update AWS links

### PR DESCRIPTION
**Pull request by carkod**
## Done

- 2 copy updates, updating AWS links
- Djlint formatting
- Refactor `_get_pro` section, so that we can reuse it in multiple pages.

## QA

- Go to https://ubuntu-com-13924.demos.haus/aws and [check copy doc comments are addressed](https://docs.google.com/document/d/1lkH-MSPmgjCMP4gI0FW29YIKVDvBuddxA63iQhmsYIU/edit#heading=h.wmtx864far5b)
- Go to https://ubuntu-com-13924.demos.haus/aws/pro and [check copy doc comments are addressed](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11844
Fixes https://warthogs.atlassian.net/browse/WD-11846

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/aws/shared/_get_pro.html [hunk 1]
```diff
@@ -5,13 +5,19 @@ <h2>Get Ubuntu Pro on AWS now:</h2>
     </div>
   </div>
   <div class="row p-divider">
-    <div class="col-4 p-divider__block">
+    <div class="col-2 p-divider__block">
+      <h3 class="p-heading--4"><a href="https://aws.amazon.com/marketplace/pp/prodview-s4zvkzmlirbga">24.04 LTS</a></h3>
```
Comment (britneywwc) [inline templates/aws/shared/_get_pro.html:9]:
```suggestion
      <h3 class="p-heading--4"><a href="https://aws.amazon.com/marketplace/pp/prodview-kbkyvwyqubv2g">24.04 LTS</a></h3>
```
The link is different from the copy doc

**Review by britneywwc — COMMENTED** [review]
LGTM, there is some link inconsistencies from the copy docs for 24.04 LTS download link, I'll address it in the copy doc.

**Comment by carkod** [comment]
> LGTM, there is some link inconsistencies from the copy docs for 24.04 LTS download link, I'll address it in the copy doc.

I believe what I did is actually correct, Carlos Bravo made the mistake. Can I get final review?

File: templates/aws/shared/_get_pro.html (hunk 1 above)
Comment (britneywwc) [inline templates/aws/shared/_get_pro.html:9]:
```suggestion
      <h3 class="p-heading--4"><a href="https://aws.amazon.com/marketplace/pp/prodview-kbkyvwyqubv2g">24.04 LTS</a></h3>
```

**Review by britneywwc — CHANGES_REQUESTED** [review]
Could you please change the link? It should be the other one

**Review by britneywwc — APPROVED** [review]
Looks good, thanks!

**Pull request merged**